tests/network/services.go

```diff
@@ -247,51 +237,35 @@ var _ = SIGDescribe("Services", func() {
 		})
 
 		Context("with a service matching the vmi exposed", func() {
-			var jobsToCleanup []func() error
-			var serviceManager *vmiServiceManager
-			var serviceNamePrefix string
-
-			BeforeEach(func() {
-				serviceNamePrefix = "myservice"
-				serviceManager = newVMIServiceManager(inboundVMI, servicePort, serviceNamePrefix, selectorLabelKey, selectorLabelValue)
-
-				services := serviceManager.buildK8sServicesSpec()
-				Expect(services).NotTo(BeEmpty(), "a service should be exposed per each iface. At least one *must* be present.")
-
-				for _, exposedService := range services {
-					_, err := virtClient.CoreV1().Services(inboundVMI.Namespace).Create(&exposedService)
-					Expect(err).NotTo(HaveOccurred())
-				}
-
-				// reset the job cleanup functions list
-				jobsToCleanup = nil
-			})
+			var jobToCleanup func() error
 
 			AfterEach(func() {
-				var errors []error
-				for _, exposedService := range serviceManager.services {
-					err := virtClient.CoreV1().Services(inboundVMI.Namespace).Delete(exposedService.Name, &k8smetav1.DeleteOptions{})
-					errors = append(errors, err)
-				}
-				for _, err := range errors {
-					Expect(err).NotTo(HaveOccurred())
-				}
+				Expect(jobToCleanup()).To(Succeed(), "cleaning up the k8sv1.Service entity should have succeeded.")
 			})
 
-			AfterEach(func() {
-				cleanupJobs(jobsToCleanup)
-			})
+			table.DescribeTable("should be able to reach the vmi based on labels specified on the vmi", func(ipFamily k8sv1.IPFamily) {
+				var service *k8sv1.Service
```

Comment:
Why do you need a variable for the service?

Reply:
So I can have a different service for each IP family, and still refer to it - https://github.com/kubevirt/kubevirt/pull/4224/files#diff-9a20ab1179984538941b03db7cae82e9R251 .